protocol/messenger_testing_utils.go

```diff
@@ -220,26 +218,6 @@ func SetIdentityImagesAndWaitForChange(s *suite.Suite, messenger *Messenger, tim
 	s.Require().True(ok)
 }
 
-func WaitForAvailableStoreNode(s *suite.Suite, m *Messenger, ctx context.Context) {
```

Comment:
I guess the functions removed here were because they were never used?

Reply:
yup, this was simply unused